package/package_esp32_index.template.json

```diff
@@ -1,188 +1,250 @@
-              "host": "arm-linux-gnueabihf",
-              "url": "https://dl.espressif.com/dl/esptool-2.6.1-linux.tar.gz",
-              "archiveFileName": "esptool-2.6.1-linux.tar.gz",
-              "checksum": "SHA-256:eaf82ff4070d9792f6a42ae1e485375de5a87bec59ef01dfb95de901519ec7fb",
-              "size": "44762"
-            }
-          ]
-        },
-        {
-          "name": "mkspiffs",
-          "version": "0.2.3",
-          "systems": [
-            {
-              "host": "i686-mingw32",
-              "url": "https://github.com/igrr/mkspiffs/releases/download/0.2.3/mkspiffs-0.2.3-arduino-esp32-win32.zip",
-              "archiveFileName": "mkspiffs-0.2.3-arduino-esp32-win32.zip",
-              "checksum": "SHA-256:b647f2c2efe6949819c85ea9404271b55c7c9c25bcb98d3b98a1d0ba771adf56",
-              "size": "249809"
-            },
-            {
-              "host": "x86_64-apple-darwin",
-              "url": "https://github.com/igrr/mkspiffs/releases/download/0.2.3/mkspiffs-0.2.3-arduino-esp32-osx.tar.gz",
-              "archiveFileName": "mkspiffs-0.2.3-arduino-esp32-osx.tar.gz",
-              "checksum": "SHA-256:9f43fc74a858cf564966b5035322c3e5e61c31a647c5a1d71b388ed6efc48423",
-              "size": "130270"
-            },
-            {
-              "host": "i386-apple-darwin",
-              "url": "https://github.com/igrr/mkspiffs/releases/download/0.2.3/mkspiffs-0.2.3-arduino-esp32-osx.tar.gz",
-              "archiveFileName": "mkspiffs-0.2.3-arduino-esp32-osx.tar.gz",
-              "checksum": "SHA-256:9f43fc74a858cf564966b5035322c3e5e61c31a647c5a1d71b388ed6efc48423",
-              "size": "130270"
-            },
-            {
-              "host": "x86_64-pc-linux-gnu",
-              "url": "https://github.com/igrr/mkspiffs/releases/download/0.2.3/mkspiffs-0.2.3-arduino-esp32-linux64.tar.gz",
-              "archiveFileName": "mkspiffs-0.2.3-arduino-esp32-linux64.tar.gz",
-              "checksum": "SHA-256:5e1a4ff41385e842f389f6b5254102a547e566a06b49babeffa93ef37115cb5d",
-              "size": "50646"
-            },
-            {
-              "host": "i686-pc-linux-gnu",
-              "url": "https://github.com/igrr/mkspiffs/releases/download/0.2.3/mkspiffs-0.2.3-arduino-esp32-linux32.tar.gz",
-              "archiveFileName": "mkspiffs-0.2.3-arduino-esp32-linux32.tar.gz",
-              "checksum": "SHA-256:464463a93e8833209cdc29ba65e1a12fec31718dc10075c195a2445b2c3f6cb0",
-              "size": "48751"
-            },
-            {
-              "host": "arm-linux-gnueabihf",
-              "url": "https://github.com/igrr/mkspiffs/releases/download/0.2.3/mkspiffs-0.2.3-arduino-esp32-linux-armhf.tar.gz",
-              "archiveFileName": "mkspiffs-0.2.3-arduino-esp32-linux-armhf.tar.gz",
-              "checksum": "SHA-256:ade3dc00117912ac08a1bdbfbfe76b12d21a34bc5fa1de0cfc45fe7a8d0a0185",
-              "size": "40665"
-            }
-          ]
-        }
-      ]
-    }
-  ]
-}
+{

```

Comment:
reformatting of an existing file (such as this one) is often discouraged and will often lead to the PR sitting for a very long time (if it ever gets merged).

If you do reformat the file, please ensure it is consistent as it doesn't appear to be consist after reformatting (looks like a mix of spaces and tabs)

Reply:
@joltwallet and @BrianPugh 
Could you please help in this?